apps/omg_watcher_rpc/lib/web/validators/merge_constraints.ex

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.Validator.MergeConstraints do
+  @moduledoc """
+  Validates `/transaction.merge` parameters
+  """
+
+  alias OMG.Utils.HttpRPC.Validator.Base
+  alias OMG.WatcherRPC.Web.Validator.Helpers
+
+  import OMG.Utils.HttpRPC.Validator.Base
+
+  require OMG.State.Transaction.Payment
+
+  defp get_constraints(params) do
+    case params do
+      %{"address" => _, "currency" => _} ->
+        [
```

Comment:
For consistency, it's better to return `{:ok, ...}` if you your func can return `{:error, ...}`